src/app/[locale]/(tabs)/user/update-password/page.tsx

```diff
@@ -0,0 +1,107 @@
+'use client';
+import {PageLayout} from '@/components';
+import {Button} from '@/components/ui/button';
+import {Input} from '@/components/ui/input';
+import {useToast} from '@/hooks/useToast';
+import {supabase} from '@/lib/supabase/supabaseClient';
+import {useTranslations} from 'next-intl';
+import {useState} from 'react';
+
+const UpdatePassword = () => {
+  const {toast} = useToast();
+  const t = useTranslations('authenticate');
+  const [formData, setFormData] = useState({
+    email: '',
+    newPassword: '',
+    confirmNewPassword: '',
+  });
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    if (formData.newPassword !== formData.confirmNewPassword) {
+      toast({
+        title: t('error'),
+        description: t('passwordsDoNotMatch'),
+        variant: 'destructive',
+      });
+      return;
+    }
+    try {
+      const {error} = await supabase.auth.updateUser({
+        email: formData.email,
+        password: formData.newPassword,
+      });
+      if (error) throw error;
+      setFormData({email: '', newPassword: '', confirmNewPassword: ''});
+    } catch (error) {
+      console.error('Error updating password:', error);
+      toast({
+        title: t('error'),
+        description: t('errorDescription'),
+        variant: 'destructive',
+      });
+    }
+  };
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const {name, value} = e.target;
+    setFormData((prevState) => ({
+      ...prevState,
+      [name]: value,
```

Comment:
I can't write anything in the inputfields. I think it's this line that causes it. When I enter something in the input fields it overwritten by the previous state which is empty. 


https://github.com/user-attachments/assets/241a0e96-1e34-4a24-9f19-6522b7dd722c